design/ceph/object-store.md

```diff
@@ -95,51 +92,16 @@ The RGW service can be configured to listen on both http and https by specifying
     instances: 1
 ```
 
-### Realms, Zone Groups, and Zones
+### Realm
+The realm settings correspond to the Ceph RGW Multisite configuration.
```

Comment:
If the `realm` is not specified in the object CR, we will get the same behavior as today where the defaults are created and no separate realm CR is needed, right? Maybe it's already in the doc somewhere and i'm not there yet...

Reply:
Probably it will be useful to define always a multisite configuration by default in the object store CR.  Something like:

```
kind: CephObjectStoreRealm
metadata:
  name: rook-realm
  namespace: rook-ceph
spec:
  bootstrap-realm:
    name: rook-realm
    zonegroups:
      rook-zonegroup-a:
        is-master: true
        master-zone: zone-z
```

Also to comment that maybe we should use just one CRD to define all the stuff around RGW. I think that the new "object-store-realm.yaml" CRD content should be included into "object-store.yaml"

Taking into account current limitations ("_zonegroups section of the object-store-realm CR is only additive, and realm can not be deleted (only through toolbox)"_). We should warn in the CRD and in the documentation about this limitations and that is needed to have a clear vision of the RGW multisite topology  in order to configure properly the object store CR.

Reply:
@travisn yes, if realm is not specified than whatever the behavior is today applies.